samples/snippets/quickstart_json_credentials_test.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.analytics.data_v1beta import BetaAnalyticsDataClient
+import quickstart_json_credentials
+
+
+def test_quickstart(capsys):
+  TEST_PROPERTY_ID = '222596558'
```

Comment:
Please don't hardcode test values - [use an environment variable instead](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#external-resources).

Reply:
I checked with @busunkim96 earlier, Bu confirmed it is currently not possible to set up environment variables for tests running on github. Could you kindly point me in the right direction perhaps?

Reply:
Updated this to use the env variable with a hardcoded default value, while we are working on setting the environment.

Reply:
Ah sorry @ikuleshov  I thought you were talking about the system tests. For sample tests there is infrastructure set up to do this: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#environment-variables

Reply:
Thank you, Bu. I have added an environment variable into noxfile_config.py and updated the tests.